ci/deploy-vmck

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash -ex
+
+cd "$( dirname "${BASH_SOURCE[0]}" )"
+
+export VAGRANT_DOTFILE_PATH=$(mktemp -d --tmpdir VAGRANT_DOTFILE_XXXXXXXX)
+export VAGRANT_DEFAULT_PROVIDER=vmck
+export VAGRANT_CHECKPOINT_DISABLE=true
+export VAGRANT_BOX_UPDATE_CHECK_DISABLE=true
+
+FILENAME=$(basename -- "$PROVISION")
+STEM=${FILENAME%.*}
+export VMNAME="$DRONE_REPO_NAME-$DRONE_BUILD_NUMBER-$STEM"
```

Comment:
What do these lines do? Maybe add a comment to help the next poor sod who wants to hack on the code :)